packages/scene-composer/src/interfaces/sceneViewer.ts

```diff
@@ -14,6 +14,18 @@ export interface SceneViewerConfig {
   locale?: string;
 }
 
+export interface MatterportConfig {
+  modelId: string;
+  accessToken?: string; // OAuth usage
+  applicationKey?: string; // Supports Demo case where the window uses an application key
```

Comment:
Demo case comment is not clear to me? an application key will always ben required when hosted on non-matterport servers